2-mandatory/2-financial-times.js

```diff
@@ -15,6 +24,16 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+    const title = [];
+    let titleWithFewestWord = allArticleTitles[0].length;
+  for (let fewWord of allArticleTitles){
+      if (fewWord.split(' ').length < titleWithFewestWord){
+          titleWithFewestWord = fewWord.split(' ').length
+          title.push(fewWord)
+      }else{
+          titleWithFewestWord = titleWithFewestWord;
```

Comment:
I don't think we need this else statement here as it's not doing anything. `titleWithFewestWord` already equals `titleWithFewestWord`. We can just stick with the if statement only